java/shared/src/test/java/com/squareup/subzero/shared/SubzeroUtilsTest.java

```diff
@@ -526,4 +610,34 @@ private TxOutput testOutput(long amount, Destination destination) {
             .setIndex(2))
         .build();
   }
+
+  private ECKey newMockPubKey() {
+    return new ECKey();
```

Comment:
does this constructor generate a new random key pair? If so, it's not really a "Mock", it's a real key. So maybe call the method `newRandomKeyPair()` instead?